arm-network/2017-03-01/swagger/virtualNetwork.json

```diff
@@ -680,6 +680,48 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/usages": {
+      "get": {
+        "operationId": "VirtualNetworks_GetUsage",
```

Comment:
Not sure if it should. Say List function for virtual networks returns
```
[
  {vnet1 details},
...
  {vnetN details}
]
```
this one returns
```
{
  "value": [
    {value1},
    ...
    {ValueN}
  ]
}
```
So root item is not an array here although it has only "value" property that is array. Should it start with "List" in this case?